### ISSUE TITLE
onNestedPreFling throws error for projects that have minSDK < 21

### ISSUE THREAD

**Issue by tomafc330** [ashqal/ChromeLikeSwipeLayout#7]
So I was testing on an older phone (api 15) and used the library and unfortunately it crashes on fling with the following stacktrace:

```
02-08 17:36:44.324 20427-20427 E/AndroidRuntime: FATAL EXCEPTION: main
                                                                      Process: com.adhoclabs.burner, PID: 20427
                                                                      java.lang.NoSuchMethodError: android.view.ViewGroup.onNestedPreFling
                                                                          at com.asha.ChromeLikeSwipeLayout.onNestedPreFling(ChromeLikeSwipeLayout.java:578)
                                                                          at android.support.v4.view.ViewParentCompat$ViewParentCompatStubImpl.onNestedPreFling(ViewParentCompat.java:124)
                                                                          at android.support.v4.view.ViewParentCompat.onNestedPreFling(ViewParentCompat.java:404)
                                                                          at android.support.v4.view.NestedScrollingChildHelper.dispatchNestedPreFling(NestedScrollingChildHelper.java:252)
                                                                          at android.support.v7.widget.RecyclerView.dispatchNestedPreFling(RecyclerView.java:8675)
                                                                          at android.support.v7.widget.RecyclerView.fling(RecyclerView.java:1797)
                                                                          at android.support.v7.widget.RecyclerView.onTouchEvent(RecyclerView.java:2437)
                                                                          at android.view.View.dispatchTouchEvent(View.java:7782)
                                                                          at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2316)
                                                                          at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2013)
```

Looks like we might need to do a version check before call `super`. for `onNestedFling` and `onNestedPreFling`.

**Comment by tommytcchan** [comment]
references #8

**Comment by tomafc330** [comment]
fixed in 0.4
